fairscale/optim/oss.py

```diff
@@ -395,3 +424,64 @@ def _free_other_grads(self) -> None:
             for p in partition:
                 for t in p["params"]:
                     t.grad = None
+
+    @staticmethod
+    def _scatter(buffer: torch.Tensor, params: List[Parameter], gradients: bool = False) -> None:
```

Comment:
I feel the gradients argument adds complexity. Why not just pass in a list of gradients instead?

Reply:
ah this is interesting, I was wondering about that when writing this down. Ok, changing this